### ISSUE TITLE
Support for Webstorm 2019.1

### ISSUE THREAD

**Issue by red2678** [GoogleCloudPlatform/google-cloud-intellij#2477]
(Please ensure you are running the latest version of Cloud Tools for IntelliJ with _Help > Check for Updates_.)

- Cloud Tools for IntelliJ version: 19.2.1
- Google Cloud SDK version _(run `gcloud version` on the command line)_:
  - Google
  - Cloud SDK 240.0.0
  - bq 2.0.42
  - core 2019.03.22
  - sutil 4.37

- OS: Windows 10 Pro

**What did you do?**
Installed the plugin on Webstorm 2019.1

Version Info for Webstorm:
- WebStorm 2019.1
- Build #WS-191.6183.63, built on March 21, 2019
- JRE: 1.8.0_202-release-1483-b39 amd64
- JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

**What did you expect to see?**
It load.

**What did you see instead?**
An error about not having the Maven and Gradel plugins installed (possibly other I am unable to tell).  When I click the "enable" link next to the missing plugins nothing happens. I believe this is because those plugins are not supported by Webstorm.

Screenshots:
![image](https://user-images.githubusercontent.com/1896036/55325501-46e5e100-5453-11e9-8d5c-67472787a256.png)

![image](https://user-images.githubusercontent.com/1896036/55325541-6a109080-5453-11e9-8f0b-82efea933116.png)

![image](https://user-images.githubusercontent.com/1896036/55325532-62e98280-5453-11e9-9ee5-88149256c912.png)

**Comment by etanshaul** [comment]
Hi. Thanks for reporting. The plugin version 19.3.1 was recently released which should span version 2019.1 (including webstorm). 

By any chance, do you have a custom plugin URL entered in the plugin settings for Cloud Tools? This could be preventing you from finding the latest version.

**Comment by red2678** [comment]
@etanshaul you are correct, I was running an older version of the plugin. When I updated Webstorm, it ported the older version of the plugin to the new version of the IDE. I never got a notification to update Cloud Tools on the latest version of Webstorm (I did my other plugins), so I wrongly assumed I was using the most recent version. I have uninstalled the plugin and reinstalled **19.3.1** and can confirm it works just fine. Thanks!
